app/assets/javascripts/initializers/initScrolling.js.erb

```diff
@@ -195,6 +195,7 @@ function algoliaPaginate(tag){
       page: nextPage,
       attributesToHighlight: [],
       tagFilters: filters,
+      filters: (requiresApproval === 'true') ? 'approved:true' : ''
```

Comment:
I wonder if we shouldn't rename the `filters` variable to `tagFilters` :)

Reply:
yesssss, this is a good point! I've pushed this change :)